feature-libs/organization/schematics/add-organization/__snapshots__/index_spec.ts.snap

```diff
@@ -1,5 +1,227 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Spartacus Organization schematics: ng-add Account summary feature eager loading should import appropriate modules 1`] = `
+"import { NgModule } from '@angular/core';
+import { I18nConfig, provideConfig } from \\"@spartacus/core\\";
+import { AccountSummaryModule } from \\"@spartacus/organization/account-summary\\";
+import { accountSummaryTranslationChunksConfig, accountSummaryTranslations } from \\"@spartacus/organization/account-summary/assets\\";
+import { OrderApprovalRootModule } from \\"@spartacus/organization/order-approval/root\\";
+
+@NgModule({
+  declarations: [],
+  imports: [
+    OrderApprovalRootModule,
+    AccountSummaryModule
```

Comment:
should be only `AccountSummaryRootModule` in the imports array.

Reply:
fixed